smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -79,9 +79,15 @@ public void generateSharedComponents(GenerationContext context) {
         TypeScriptWriter writer = context.getWriter();
 
         SymbolReference responseType = getApplicationProtocol().getResponseType();
-        writer.addImport("ResponseMetadata", "ResponseMetadata", "@aws-sdk/types");
-        writer.openBlock("const deserializeMetadata = (output: $T): ResponseMetadata => ({", "});", responseType,
-                () -> {
+        SymbolReference responseMetadataType = SymbolReference.builder()
```

Comment:
You could just change line 82's second parameter to "__ResponseMetadata" and then manually refer to it using the alias in code.